x-pack/plugins/runtime_fields/public/components/runtime_field_editor/runtime_field_editor.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { DocLinksStart } from 'src/core/public';
+
+import { RuntimeField } from '../../types';
+import { getLinks } from '../../lib';
+import { RuntimeFieldForm, Props as FormProps } from '../runtime_field_form/runtime_field_form';
+
+export interface Props {
+  docLinks: DocLinksStart;
+  defaultValue?: RuntimeField;
+  onChange?: FormProps['onChange'];
+}
+
+export const RuntimeFieldEditor = ({ defaultValue, onChange, docLinks }: Props) => {
+  const links = getLinks(docLinks);
+
+  return <RuntimeFieldForm links={links} defaultValue={defaultValue} onChange={onChange} />;
```

Comment:
For now, this component only renders the `<RuntimeFieldForm />` but in a later stage it will also render the "Preview field" functionality.